docs/sources/reference/components/prometheus.exporter.blackbox.md

```diff
@@ -40,6 +49,15 @@ The `config` argument must be a YAML document as string defining which blackbox_
 - `remote.http.LABEL.content`
 - `remote.s3.LABEL.content`
 
+The `targets` argument is an alternative to the [target][] block. This is useful when blackbox targets are supplied by another component.
+The following labels can be set to a target:
```

Comment:
```suggestion
You can set the following labels to a target:
```

Do we add the labels or set the labels?

Small change to make it more active. It's OK as-is too... this is just a small step towards making our docs active vs passive :-)

Reply:
I will use add, it seems slightly better